### ISSUE TITLE
Git History Diff 2.3.1  doesn't work on remote-ssh env

### ISSUE THREAD

**Issue by litianhe** [huizhougit/githd#58]
# Description
I have a remote linux env,  and I use VSCode remote-ssh plugin to connect to it.   I also installed `Git History Diff ` in the remote linux. Tody  this plug doesn't work fine, and the version is upgrade to v2.3.1.   

Then I choose to install another version v2.2.4 and it's back to normal.

## My VSCode version:
```
Version: 1.69.1 (user setup)
Commit: b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a
Date: 2022-07-12T08:21:24.514Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64 10.0.19042
```

## My remote linux env:  
```
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
```

# Error log on Remote Server:
```
2022-07-18 19:01:27.869] [remoteagent] [info] Getting Manifest... huizhou.githd
[2022-07-18 19:01:29.843] [remoteagent] [info] Installing extension: huizhou.githd
[2022-07-18 19:01:31.812] [remoteagent] [error] Error while installing the extension: huizhou.githd
[2022-07-18 19:01:31.813] [remoteagent] [error] Failed to install extension. huizhou.githd
[2022-07-18 19:01:31.814] [remoteagent] [error] Download: self signed certificate in certificate chain
    at x.downloadExtension (/home/mystic/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/server/node/server.main.js:137:87197)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async x.doRun (/home/mystic/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/server/node/server.main.js:137:86318)
```

If you need more logs/actions please let me know.
It's one of my favorite VSCode plugin, much appreciate if this issue can be fixed.

**Comment by chengzhang-90** [comment]
俺也一样

**Comment by huizhougit** [comment]
fixed in v2.3.2
